rosys/hardware/gnss.py

```diff
@@ -140,9 +141,9 @@ async def _run(self) -> None:
         last_num_satellites = 0
         last_hdop = 0.0
         last_altitude = 0.0
-        last_gga_timestamp = ''
-        last_gst_timestamp = ''
-        last_pssn_timestamp = ''
+        last_gga_timestamp = 0.0
+        last_gst_timestamp = 0.0
+        last_pssn_timestamp = 0.0
```

Comment:
When changing these variable types, please update the line
```py
if last_gga_timestamp == last_gst_timestamp == last_pssn_timestamp != '':
```
accordingly. I guess we need to compare with 0.0 instead.